src/remotelookup.h

```diff
@@ -42,7 +42,7 @@ class Inbound;
 /**
  *  Class definition
  */
-class RemoteLookup : public Lookup, public std::enable_shared_from_this<RemoteLookup>, private Processor, private Connector
+class RemoteLookup : public Lookup, public std::enable_shared_from_this<RemoteLookup>, private Processor, public Connector
```

Comment:
Note: this had to be done because otherwise `shared_from_this()` wasn't convertible to an `std::shared_ptr<Connector>`